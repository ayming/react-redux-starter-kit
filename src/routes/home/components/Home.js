import React, { PureComponent } from 'react'

import Link from '../../../components/Link'
import s from './Home.scss'

class Home extends PureComponent {
  render() {
    return (
      <div className={s.wrapper}>
        <p>
          To get started, edit <code>src/routes/home/components/Home.js</code>{' '}
          and save to reload.
        </p>
        <Link to="/about">ABOUT</Link>
      </div>
    )
  }
}

export default Home
