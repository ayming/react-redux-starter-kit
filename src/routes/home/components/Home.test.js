/* eslint-env jest */
import React from 'react'
import renderer from 'react-test-renderer'

import Home from './Home'

describe('Home', () => {
  test('renders correctly', () => {
    const wrapper = renderer.create(<Home />).toJSON()

    expect(wrapper).toMatchSnapshot()
  })
})
