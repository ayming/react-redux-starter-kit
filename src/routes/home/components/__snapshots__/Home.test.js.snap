// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders correctly 1`] = `
<div
  className="wrapper"
>
  <p>
    To get started, edit 
    <code>
      src/routes/home/components/Home.js
    </code>
     
    and save to reload.
  </p>
  <a
    href="/about"
    method="push"
    onClick={[Function]}
  >
    ABOUT
  </a>
</div>
`;
